models/errors/model-errors.js

```diff
@@ -1,16 +1,20 @@
-exports.resourceError = (id, type) => {
+resourceError = (resource, property, value) => {
   return Promise.reject({
     status: 404,
-    msg: `There is no ${type} with the id ${id}`,
+    msg: `There is no ${resource} with the ${property} ${value}`,
   });
 };
 
-exports.dbError = (err, msg) => {
+dbError = (err, resource, property, value) => {
   if (err.code === "22P02") {
```

Comment:
🚨 danger!

You are declaring global variables here! 😱

you *must* use `let` or `const` keywords

Reply:
Whoops! Can't believe I didn't spot that. Perhaps need to get a linter installed..